packages/flutter/lib/src/animation/tween.dart

```diff
@@ -68,6 +78,8 @@ class Tween<T extends dynamic> extends Animatable<T> {
   T lerp(double t) => begin + (end - begin) * t;
 
   /// Returns the interpolated value for the current value of the given animation.
+  ///
+  /// This method returns `begin` and `end` when the animation values are 0.0 or 1.0, respectively.
```

Comment:
I think you took this from my docs, but the paraphrasing is missing the important point. An interpolation of double should (seemingly) always return begin for 0.0 and end for 1.0. However, it's a base 2 floating point operation, which means that some numbers will be approximately calculated. By returning the begin and end variables as a guarantee, it simplifies the contract with the caller.